doc/memos/rdm0.md

```diff
@@ -0,0 +1,153 @@
+authors,
+2. a section with the main references that relate to the content, with a strong
+preference for permanent, self-contained references,
+3. the list of revisions so far and corresponding changelog,
+4. contact information to reach the author(s).
+
+
+# 3. Memo Publishing and Maintenance Process
+
+Each new memo MUST be proposed as a pull-request (PR),
+and discussed the same way RIOT code PRs are processed on GitHub.
+
+Initially, a proposed RDM is identified by a temporary name 
+similar to IETF Internet Draft names (for
+example: rdm-draft-baccelli-rdm-format.md) instead of a number
+assigned at a later stage.
+
+Proposing an RDM is non-compulsory. There is no restriction on when, in the 
+development cycle, an RDM can be PRed. However, RDMs on design decisions and 
+architecture are typically expected to describe something rather stable (that is 
+already merged in the main branch of RIOT, or about to be merged).
+
+Once proposed, a memo MUST have received an ACK by at least 3 maintainers other 
+than the author(s) of the memo, before its publication. 
+
+At this stage, the RDM is assigned a permanent RDM number (the next available),
+and merged in the master branch of RIOT. Once merged into the master branch
+of RIOT, a memo is considered published.
+
+Small changes/clarifications may be subsequently PRed to improve a memo, using 
+the same RDM number. In this case, a (monotically increasing) revision number 
+is issued and a changelog is added -- see Revision section in postamble below.
+
+Bigger changes should result in a new memo with another 
+RDM number, deprecating the old memo. In this case, the deprecated memo must 
+be modified to append a line in its header indicating "DEPRECATED BY RDM X" 
+where X is the RDM number of the new memo.
+
+A dedicated directory in RIOT codebase (RIOT/doc/memos) contains all the memos.
+It is acceptable to add graphical content (in RIOT/doc/memos/graphics/) to
+illustrate the memo, as done with the RIOT logo below (see Figure 1, given here
+for pure sake of example). However graphical content should be kept to the bare 
+minimum, as it is easy to get lost in "shiny slideware".
+
+
+![Figure 1. RIOT logo.](graphics/rdm0/RIOT-logo.png "RIOT logo")
+<p align="justify">
+*Figure 1. The RIOT logo.*
+</p>
+
+
+## Acknowledgements
+
+Thanks to Thomas Eichinger, Oliver Hahm, Ludwig Knüpfer, Martine Lenders, 
+Sebastian Meiling, Hauke Petersen, Dan Petry, Kaspar Schleiser and
+Matthias Wählisch for their comments and suggestions.
+
+
+## References
+
+
+- Internet Engineering Task Force ([IETF](http://ietf.org/))
+- Guide to [Contributing to RIOT](https://github.com/RIOT-OS/RIOT/wiki/Contributing-to-RIOT)
+
+## Revision
+
+- Rev2: added title in metadata and specification of temporary RDM naming
+- Rev1: added text on non-compulsory nature of RDMs
+- Rev0: initial document
+
+## Contact
+
+The author of this memo can be contacted by email at emmanuel.baccelli@inria.fr
```

Comment:
Pretty sure the wording should be "via email", but English is not my native language ;-).

Reply:
I usually say via, but probably both are grammatically correct

Reply:
P.S. we could perhaps start using something like [this](https://www.grammarly.com/) to check documents and make them read more professional-like

Reply:
Maybe rather use something that is integrated in the GitHub check system. AFAIK, grammarly is not.

Reply:
yeah sure. I was more thinking that people could use it themselves to assist manual checking.